src/tools/pontoon/glossary.md

```diff
@@ -35,6 +35,15 @@ Users in Pontoon can have different permissions depending on their role:
 * Translators can either submit approved translations directly or review suggestions from other users. Unlike in some other Translation Management Systems, this is a single permission in Pontoon, and it’s referred to as *Translator permission*.
 * Team managers have translator permissions, but can also change permissions assigned to other users (i.e., ”promote” or ”demote” them).
 
+## Pretranslation
+
+Pretranslation is a feature in Pontoon that relies on machine translation ([Google AutoML Translation](https://cloud.google.com/translate/automl/docs)) and [translation memory](#translation-memory) to automatically translate strings and save them in localized files.
+
+If pretranslation is enabled for a combination of locale and project, when a new string is added in Pontoon:
```

Comment:
Maybe "combination of _a_ locale and _a_ project" or "a project-locale combination" is more accurate?

Reply:
Thanks, I'll go with your first suggestion.